src/customers/account-dashboard-gift-registry.md

```diff
@@ -7,3 +7,51 @@ The [Gift Registry]({% link marketing/gift-registries.md %}) section of the cust
 
 ![]({% link images/images-ee/account-dashboard-gift-registry.png %}){: .zoom}
 _Gift Registry_
+
+## To create new gift registry
```

Comment:
Create a new gift registry

Reply:
Hi @danidelcar, thank you for reviewing!

I have implemented this suggestion!